cfgov/cfgov/settings/base.py

```diff
@@ -501,3 +501,8 @@
 
 # Akamai
 ENABLE_AKAMAI_CACHE_PURGE = os.environ.get('ENABLE_AKAMAI_CACHE_PURGE', False)
+if ENABLE_AKAMAI_CACHE_PURGE:
+    AKAMAI_USER = os.environ['AKAMAI_USER']
```

Comment:
I *could* update these to `os.environ.get()` here, and then throw an error in `flush_akamai` if they are `None`. A little verbose, but perhaps that's the best compromise?